### DISCUSSION TITLE
Exposure / Flickering problem during night vision, any solution possible with the yi hacks?

### DISCUSSION THREAD

**Discussion by borgqueenx** [roleoroleo/yi-hack-Allwinner-v2#764] [General]
I have a big problem with night vision with my yi cams, i bought 5 during black friday, but might need to return all of them :(
The problem is that night vision is constantly switching exposure, like in this video:
https://www.youtube.com/watch?v=o9EGKB2U5fQ
Seems a lot of people have this issue. a way to make this less aggresive or only allow once in x minutes would be great!

Hope something can be done about this...

**Comment by roleoroleo** [comment]
Yes, it's a common problem. One of my cams has this problem too.
And I didn't find a solution.
